target/linux/ramips/dts/ApexeraNetBot.dts

```diff
@@ -0,0 +1,183 @@
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "apexera-netbot", "ralink,mt7620a-soc";
+	model = "Apexera NetBot";
+
+	chosen {
+		bootargs = "console=ttyS1,57600";
+        };
+
+	gpio-leds {
+		compatible = "gpio-leds";
+
+		wifi {
+			label = "netbot:led:wifi";
+			gpios = <&gpio2 27 1>;
+		};
+
+		wan {
+			label = "netbot:led:wan";
+			gpios = <&gpio2 26 1>;
+		};
+
+		usb2g {
+			label = "netbot:green:usb2";
+			gpios = <&gpio2 1 1>;
+		};
+
+		usb2r {
+			label = "netbot:red:usb2";
+			gpios = <&gpio2 0 1>;
+		};
+
+		usb4g {
+			label = "netbot:green:usb4";
+			gpios = <&gpio2 3 1>;
+		};
+
+		usb4r {
+			label = "netbot:red:usb4";
+			gpios = <&gpio2 2 1>;
+		};
+
+		usb1g {
+			label = "netbot:green:usb1";
+			gpios = <&gpio2 21 1>;
+		};
+
+		usb1r {
+			label = "netbot:red:usb1";
+			gpios = <&gpio2 22 1>;
+		};
+
+		usb3g {
+			label = "netbot:green:usb3";
+			gpios = <&gpio0 22 1>;
+		};
+
+		usb3r {
+			label = "netbot:red:usb3";
+			gpios = <&gpio0 23 1>;
+		};
+	};
+};
+
+&i2s {
+	status = "okay";
+};
+
+&gpio1 {
+	status = "okay";
+};
+
+&gpio2 {
+	status = "okay";
+};
+
+&uart {
+	status = "okay";
+};
+
+&i2c {
+	status = "okay";
+
+	pcf8563: rtc@51 {
+		status = "okay";
+		compatible = "nxp,pcf8563";
+		reg = <0x51>;
+	};
+
+	wm8978: wm8978@1A {
+		compatible = "wm8978";
+		reg = <0x1A>;
+	};
+};
+
+&spi0 {
+	status = "okay";
+	#address-cells = <1>;
+	#size-cells = <0>;
+
+	m25p80@0 {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		compatible = "w25q128";
```

Comment:
has to be `compatible = "jedec,spi-nor";` check the other dts files for reference.